specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -157,7 +157,21 @@
             "required": true,
             "type": "string",
             "enum": ["daily", "monthly"],
-            "modelAsString":true
+            "x-ms-enum": {
+              "name": "datagrain",
+              "modelAsString":true,
+              "values":[
+                        { 
```

Comment:
minor: formatting/indenting seems off, if you could use 2 spaces for formatting it'd be great.

Reply:
Done